stable/yugaware/templates/_helpers.tpl

```diff
@@ -112,3 +112,16 @@ Make list of allowed CORS origins
 {{- end -}}
 ]
 {{- end -}}
+
+{{/*
+kube-state-metrics
+*/}}
+{{- define "kubeStateMetric.endpoint" -}}
```

Comment:
```suggestion
{{- define "yugaware.kubeStateMetrics.endpoint" -}}
```

> A popular naming convention is to prefix each defined template with the name of the chart: {{ define "mychart.labels" }}.

https://helm.sh/docs/chart_template_guide/named_templates/#declaring-and-using-templates-with-define-and-template